test/extensions/resource_monitors/cgroup_memory/cgroup_memory_monitor_test.cc

```diff
@@ -12,12 +18,19 @@ namespace ResourceMonitors {
 namespace CgroupMemory {
 namespace {
 
-class MockCgroupMemoryStatsReader : public CgroupMemoryStatsReader {
+// Mock filesystem implementation for testing.
+class MockFilesystem : public Filesystem::Instance {
```

Comment:
Any reason to not just re-use https://github.com/envoyproxy/envoy/blob/d1f228d4e6b06cca9304984c9484769b2fc7fc22/test/mocks/filesystem/mocks.h#L56 ?